ur_robot_driver/config/ur10_controllers.yaml

```
@@ -130,6 +130,36 @@ joint_group_vel_controller:
    type: velocity_controllers/JointGroupVelocityController
    joints: *robot_joints
 
+forward_joint_traj_controller:
+    type: "joint_trajectory_controllers/PassThroughController"
+    joints: *robot_joints
+
+forward_cartesian_traj_controller:
+    type: "cartesian_trajectory_controllers/PassThroughController"
+    joints: *robot_joints
+
+twist_controller:
+  type: "cartesian_ros_controllers/TwistController"
+  frame_id: "tool0_controller"
+  publish_rate: *loop_hz
+  joints: *robot_joints
+
+pose_traj_controller:
```

```suggestion
pose_based_cartesian_traj_controller:
```
More verbose but less confusing for users.